Formula/v8.rb

```diff
@@ -113,10 +136,57 @@ def install
     system "gn", "gen", "--args=#{gn_args_string}", "out.gn"
     system "ninja", "-j", ENV.make_jobs, "-C", "out.gn", "-v", "d8"
 
-    # Install all the things
+    # Install libraries and headers into libexec so d8 can find them, and into standard directories
+    # so other packages can find them and they are linked into HOMEBREW_PREFIX
     (libexec/"include").install Dir["include/*"]
-    libexec.install Dir["out.gn/lib*.dylib", "out.gn/d8", "out.gn/icudtl.dat"]
+    include.install_symlink Dir["#{libexec}/include/*"]
+
+    libexec.install Dir["out.gn/d8", "out.gn/icudtl.dat"]
     bin.write_exec_script libexec/"d8"
+
+    libexec.install Dir["out.gn/#{shared_library("*")}"]
+    lib.install_symlink Dir["#{libexec}/#{shared_library("*")}"]
+
+    # Install .pc files so libraries and headers can be found with pkg-config
+    (lib/"pkgconfig/v8.pc").write <<~EOS
+      prefix=#{prefix}
+      exec_prefix=${prefix}
+      libdir=${prefix}/lib
+      includedir=${prefix}/include
+
+      Name: v8
+      Description: V8 JavaScript Engine
+      Version: #{version}
+      Requires: icu-uc, icu-i18n, v8_libbase = #{version}
+      Libs: -L${libdir} -lv8 -pthread
+      Cflags: -I${includedir}
+    EOS
+
+    (lib/"pkgconfig/v8_libbase.pc").write <<~EOS
+      prefix=#{prefix}
+      exec_prefix=${prefix}
+      libdir=${prefix}/lib
+      includedir=${prefix}/include
+
+      Name: v8
+      Description: V8 JavaScript Engine - Base library
+      Version: #{version}
+      Libs: -L${libdir} -lv8_libbase
+      Cflags: -I${includedir}
+    EOS
+
+    (lib/"pkgconfig/v8_libplatform.pc").write <<~EOS
+      prefix=#{prefix}
+      exec_prefix=${prefix}
+      libdir=${prefix}/lib
+      includedir=${prefix}/include
+
+      Name: v8
+      Description: V8 JavaScript Engine - Platform library
+      Version: #{version}
+      Libs: -L${libdir} -lv8_libplatform -pthread
+      Cflags: -I${includedir}
+    EOS
```

Comment:
If upstream doesn't do this, we don't want to do it either. Maybe someone can make a PR for this upstream?

Reply:
Yes, let's not ship our own `pc` files unless we absolutely really need them.

Reply:
I removed the `pc` files based on this feedback.  It looks like the easiest way to try to get that added upstream is to open an issue on their bug tracker and find out if this requires more changes to the build system than just adding the files.